### PR TITLE
Removed Beta tag from Bulk, Batch and Patch APIs. Deleted Deprecated classes

### PR DESCRIPTION
**Pull request by kushagraThapar**
* Removed Beta tag from Bulk, Batch and Patch APIs. 
* Deleted Deprecated classes

### CODE REVIEW

**Review by xinlian12 — APPROVED** [review]
LGTM. Thanks

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkOperationResponse.java [hunk 1]
```diff
@@ -54,22 +52,34 @@
...
+     * Get Exception
+     * @return Exception
+     */
     public Exception getException() {
         return exception;
     }
 
-    @Beta(value = Beta.SinceVersion.V4_19_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    /**
+     * Get BatchContext
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkOperationResponse.java:80]: Do we need to add more detailed comment?

Comment (xinlian12) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkOperationResponse.java:80]: do we need more detailed comments?

**Review by xinlian12 — CHANGES_REQUESTED** [review]
Request change to block from merging.

**Review by milismsft — APPROVED** [review]

**Review by xinlian12 — APPROVED** [review]

**Review by moderakh — APPROVED** [review]
LGTM. thanks @kushagraThapar

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

**Pull request merged**